README.markdown

```diff
@@ -8,6 +8,9 @@ Database Cleaner is a set of gems containing strategies for cleaning your databa
 The original use case was to ensure a clean state during tests.
 Each strategy is a small amount of code but is code that is usually needed in any ruby app that is testing with a database.
 
+**NB** Please note that this README is not of the gem's latest version, you may
```

Comment:
This should say `this README is for the unreleased development version` as its technically the latest version, just not the last release.